specification/implementation/metadata/mets/mets-root/index.md

```diff
@@ -1,7 +1,9 @@
 ### 5.3.1.	Use of the METS root element (element mets)
 The purpose of the METS root element is to describe the container for the information being stored and/or transmitted. The implementation of the root element for a METS document conformant with CSIP uses attributes from the METS specification and attributes added for the purposes of the CSIP.
 
-In addition to the attributes the METS root element mets MUST define all relevant namespaces and locations of XML schemas using the `@xmlns` and `@xsi:schemaLocation` attributes.
+In addition to its attributes the METS root element mets MUST define all relevant namespaces and XML schema locations used in the package employing the @xmlns and @xsi:schemaLocation attributes.
+
+When implementing and using XML schemas consideration of physical placement of the schemas needs to be considered taking into account unaccessability of the external resource which might host the XML schema used for validation.
```

Comment:
Sorry, left out the 2nd sentence.

When implementing and using XML schemas the physical location of any schemas needs to be considered accounting for potential unavailability of any resources required for validation that are hosted externally.